modules/nw-proxy-configure-object.adoc

```diff
@@ -24,13 +22,11 @@ spec:
 status:
 ----
 
-A cluster administrator can configure the proxy for {product-title} by modifying
-this `cluster` Proxy object.
+A cluster administrator can configure the proxy for {product-title} by modifying this `cluster` `Proxy` object.
```

Comment:
```suggestion
A cluster administrator can configure the `Proxy` for {product-title} by modifying this `cluster` `Proxy` object.
```

Reply:
This one I interpreted the first "proxy" as a general term, not the object, so it would be lc. (This is per the previous reviewer's ask. These fixes outside the footnote are out of the scope of this PR, so I don't want to change any language on these extras.) https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting